packages/ui/src/Toggle.module.scss

```diff
@@ -0,0 +1,138 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+$h: c.$grid * 6;
+$w: c.$grid * 15;
+$labelMargin: c.$grid * 4;
+$trackBr: 22px;
+$trackBgOn: #1dbc22;
```

Comment:
Let's use `$colorSuccessLight` for consistency with our color palette. We'll ask @pawelklasa to verify once we implment it.